### DISCUSSION TITLE
Pre-release: normalisation model, plot improvements, bug fixes

### DISCUSSION THREAD

**Discussion by vitkl** [BayraktarLab/cell2location#61] [Q&A]
This is the last pre-release before we introduce cell2location based on pyro and scvi-tools. The main purpose of this release is fixing package versions and dependencies for pymc3 implementation of models - which helps to provide docker image that reproducibly works with older models and older user interface (run_cell2location / run_regression ...). 

This pre-release includes 1) the model which can normalise within experiment variation in RNA detection sensitivity, 2) improvements to data visualisation, 3) major refactoring of how models are organised, 4) bug fixes (no critical bugs).

<hr /><em>This discussion was created from the release <a href='https://github.com/BayraktarLab/cell2location/releases/tag/v0.06-alpha'>Pre-release: normalisation model, plot improvements, bug fixes</a>.</em>

**Comment by xiandxing** [comment]
Hello, I have a question for this pre-release usage. I have installed cell2location `v0.6a0` on server A long time ago, but when I pip install `cell2location-0.06-alpha.tar.gz` on server B now, the version of cell2location turns to `0.5`. Could you please help me? What's the problem?  THX!
